app/boards/shields/leeloo/README.md

```diff
@@ -1,41 +1,149 @@
-- Support for MX/Box or Low Profile Choc switches.
-- 90% of the switches are socketed; with the exception to the rotary encoder positions—6 positions require soldering.
-- Support for 128x32 OLED Displays.
-- The option to select one of three positions for an EC11 rotary encoder on each half.
-- Support for Alps Alpine Micro Switch
-- Support for 3.7v 301230 LiPo Battery
+- Support for both Low Profile Choc switches, and Box/MX switches; 19.05mm x 19.05mm spacing.
+- 90% of the switches are socketed; with the exception to the rotary encoder positions.
+- Support for Alps Alpine EC11 Rotary Encoders—one on each side, in one of three locations.
+- Support for OLED Displays or nice!view Displays.
+  - nice!view displays require a wire to be soldered from the CS Pin on nice!view display to P0.22 or D4 on the nice!nano.
+- Support for both 110mAh or 700mAh batteries.
+- Solder pads for battery leads.
+- Support for Alps Alpine Micro On/off switches.
 
-# Building Your Firmware
+# Building Leeloo's ZMK Firmware
 
 ZMK Firmware: [Introduction to ZMK](https://zmk.dev/docs/)
 Installation: [Installing ZMK](https://zmk.dev/docs/user-setup)
 Customization: [Customizing ZMK](https://zmk.dev/docs/customization)
 Development Environment: [Basic Setup](https://zmk.dev/docs/development/setup)
 
-Build command for the default keymap of Leeloo:
+Build commands for the default keymap of Leeloo v1:
+
+```
+west build -d build/left -p -b nice_nano_v2 -- -DSHIELD=leeloo_left
+west build -d build/right -p -b nice_nano_v2 -- -DSHIELD=leeloo_right
+```
+
+Build commands for the default keymap of Leeloo v2:
+
+```
+west build -d build/left_v2 -p -b nice_nano_v2 -- -DSHIELD=leeloo_rev2_left
+west build -d build/right_v2 -p -b nice_nano_v2 -- -DSHIELD=leeloo_rev2_right
+```
+
+Build commands for your custom keymap of Leeloo v1:
+
+```
+west build -d build/right -p -b nice_nano_v2 -- -DSHIELD=leeloo_right -DZMK_CONFIG="C:/dev/zmk/[yourNmae]/leeloo/config"
+west build -d build/left -p -b nice_nano_v2 -- -DSHIELD=leeloo_left -DZMK_CONFIG="C:/dev/zmk/[yourName]/leeloo/config"
+```
+
+Build commands for your custom keymap of Leeloo v2:
+
+```
+west build -d build/right_v2 -p -b nice_nano_v2 -- -DSHIELD=leeloo_rev2_right -DZMK_CONFIG="C:/dev/zmk/[yourNmae]/leeloo_v2/config"
+west build -d build/left_v2 -p -b nice_nano_v2 -- -DSHIELD=leeloo_rev2_left -DZMK_CONFIG="C:/dev/zmk/[yourName]/leeloo_v2/config"
+```
+
+## Building Leeloo's ZMK Firmware with nice!view Displays
+
+There are a couple of files that need to be adjusted before the build commands can be run.
+
+### Edit the leeloo[_rev2].keymap File
+
+Near the top 3rd of the leeloo[_rev2].keymap file, locate the following code block:
+
+```
+//nice_view_spi: &spi0 {
+//  compatible = "nordic,nrf-spim";
+//  pinctrl-0 = <&spi0_default>;
+//  pinctrl-1 = <&spi0_sleep>;
+//  pinctrl-names = "default", "sleep";
+//  cs-gpios = <&pro_micro 4 GPIO_ACTIVE_HIGH>;
+//};
+```
+
+Remove the forward slashes to resemble the following:
```

Comment:
Is simply the CS override, i.e. below, sufficient?
```dts
nice_view_spi: &spi0 {
    cs-gpios = <&pro_micro 4 GPIO_ACTIVE_HIGH>;
};
```

(Also applies to the micro)